Source/Data/Transform/KTFrequencySpectrumFFTW.cc

```diff
@@ -61,45 +61,50 @@ namespace Katydid
         //KTINFO(fslog, "neg freq offset: " << fLeftOfCenterOffset);
     }
 
+    // I have doubts about this function
     KTFrequencySpectrumFFTW::KTFrequencySpectrumFFTW(std::initializer_list<double> value, size_t nBins, double rangeMin, double rangeMax, bool arrayOrderIsFlipped) :
```

Comment:
@nsoblath I do not think that this function does what it is supposed to do. Either that, or I do not understand its purpose. From the signature it looks like it should enable initialization like 
`KTFrequencySpectrumFFTW s({0.0, 1.0, 2.0, 3.0}, 2, 0.0, 1.0);`
and that should result in a 2 bin spectrum with s[0] = 0.0 + 1.0i, s[1] = 2.0 + 3.0i. If that is the case, it is not what the function does.